### ISSUE TITLE
Deployment

### ISSUE THREAD

**Issue by kousu** [kousu/radiobrowser-server#2]
1. [ ] Rent some hosting space
2. [ ] Deploy this app with an empty database
3. [ ] (optional) Deploy https://github.com/segler-alex/radiobrowser too?
    - we should probably change the colours to not step on too many toes
5. [ ] (optional) Add a CI script that:
    - `on: release`, builds an uploads the latest `.deb`
6. [ ] (optional) Tweak deployment to use our releases; i.e. this line:
    https://github.com/kousu/radiobrowser-server/blob/6d10ed4b4a406d3069fe5e621257701d8f49cc53/ansible/roles/radiobrowser/tasks/main.yml#L22
    Where does `{{release}}` come from? Can we make it get auto-extracted from the most recent release? I think `git describe --tags --abbrev=0` is the right command.

---

What do we think about

4. [ ] Deploy a MySQL web client?

In https://github.com/kousu/radiobrowser-server/issues/1#issuecomment-804742588 @louis77 suggested giving SQL access for power-mods to dig through stuff. I thought it would be useful to allow us to accept edits before we sorted out merging with a forked database, but maybe that would be jumping the gun. I'm leaning now towards the idea that if people want to search the database they should do it with `git grep` or by loading it locally into sqlite and using [sqlitebrowser](https://sqlitebrowser.org/) (we can provide scripts for that) instead of worrying about hosting -- and securing -- a live service. Doing mass-updates is harder, but if we additionally had a sqlite-<->json converter it would be okay.

**Comment by DIWesser** [comment]
If we don't do a web client right away, how should I make sure I'm not creating a conflicting copy if I'm editing locally? Should we just try keeping an issue running where someone editing can let everyone know when they're messing with it until we have something serious in place?

Or should I just chill until y'all have sorted out git integration and have time to setup mod access?

**Comment by kousu** [comment]
Yeah, I think if you're good to chill a bit more I want to wait for louis's opinion. Is it more work to put up a web client and then switch it over to github or to get github working first? It's hard to say. The first thing we need is to buy some hosting and replicate the current site though.

**Comment by DIWesser** [comment]
Sounds good. Let me know if you want me to chip in for hosting.

**Comment by louis77** [comment]
I've created a DO team and invited @kousu @DIWesser so we have a computing space to work with. Please feel free to create resources you feel are necessary. DO also allows three free static website deployments in the new App platform which might be useful if we want to get a website up and running quickly.

I've also transferred the domain tunerapp.org into the DO account. Feel free to create the records you want. There is only one A record in use right now (gemini.tunerapp.org) which is a host serving the radio-browser directory in the Gemini space (in case you are interested: gemini://gemini.tunerapp.org - if you don't have a Gemini client, use https://github.com/skyjake/lagrange).

We can disucss funding later when we reached a point where we have something that acutally works.
